doc/api/process.md

```diff
@@ -918,6 +922,8 @@ console.log(process.env.test);
 // => 1
 ```
 
+*Note*: `process.env` is read-only in [`Worker`][] threads.
```

Comment:
My opinion only but I prefer we leave out `*Note*:` in nearly all cases.

Reply:
I believe `*Note*:` prefixes were eradicated earlier from the docs.